docs/_book.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+upper_tabs:
+  # Tabs left of dropdown menu
+  - include: /_upper_tabs_left.yaml
+  - include: /api_docs/_upper_tabs_api.yaml
+  # Dropdown menu
+  - name: Resources
+    path: /resources
+    is_default: true
+    menu:
+      - include: /resources/_menu_toc.yaml
+    lower_tabs:
+      # Subsite tabs
+      other:
+        - name: Examples
+          contents:
+            - title: Triplet Loss
+              path: /addons/examples/losses_triplet
```

Comment:
@MarkDaoust @lamberta If possible, we would like to keep our example directory at the root of the project. I'm unsure of how this linking works though -- is this a new directory that will be created on the server host?

Reply:
Hmmm, after reading the comment about using the pip installed package, I'm thinking we may have to move examples inside of `/tensorflow_addons/` ?

Reply:
Hi Sean,
If the examples/ directory contain notebook tutorials meant for the site, then no, they need to be in the docs site directory (we pull in the entire directory so relative links "across the boundary" won't work).
Also, to be consistent with the rest of the site, put these in a /tutorials directory. "Examples" has another meaning (more for demo apps)